projects/canopy/src/lib/table/table-cell/table-cell.component.scss

```diff
@@ -1,14 +1,11 @@
 @import '../../../styles/mixins.scss';
 
+$full-size-breakpoints: 'sm', 'md', 'lg' !default;
```

Comment:
I considered whether instances of these should be consolidated into a single scss file within `lib/table` but couldn't find a precedent for a similar approach. Please let me know if you think I should do this.

Reply:
I think it could probably go in an even more global place so that we can use the same breakpoints everywhere?

Reply:
@elenagarrone If there are other similar variables a global approach makes sense, otherwise I think within /lib/table would be better

Reply:
The radio also uses something similar`$stack-breakpoints: 'sm', 'md', 'lg' !default;`. 
Either you put `$full-size-breakpoints` in `/lib/table` and then once I modify the radio button I move it or you just move it now to a more global place and then I'll modify the radio variable name in a separate PR - together with some more bits to make it mobile friendly.

I don't mind which approach to be honest.

Reply:
Should they be the same? I'm just wondering if it's chance that on this occasion they happen to have the same properties.

> Nice one @pauleustice 👍
> 
> For everyone else, I used `stack` as an input for the radio buttons component but Paul rightly pointed out that that's not really mobile friendly. I'll be updating the radio component in a separate PR so that it's consistent with this approach.

@elenagarrone are you able to elaborate on that a bit? Do you mean you'll be changing the name of the input or is there more to it? I didn't quite pick up why it's not very mobile friendly.

Reply:
@thatguynamedandy I think they should be kept separate, personally.

It does raise an interesting point that I considered when assessing the approach to take with this component. This method of using existing breakpoint names/sizes keeps things standardised (and in turn matches the grid layout and other media query-related styles), but on the other hand it doesn't allow the consumer to specify precise values based on their needs, and only supports one value.

My initial thinking before Elena told me about the existing radio stack approach was to make use of a `ResizeObserver` that listened to breakpoints specified within the input, but this was a much larger piece of work to consider. It would likely need a Canopy resize observer module that certain components (table and radio group at present) would depend on. Furthermore it isn't supported by IE.

It could have looked something like `<table lgTable [fullWidthAt]="{ sm: true, md: false }">` which would be nice. Maybe to consider in future?

Reply:
Moved to `lib/table/table.breakpoints.scss`